src/index.tsx

```diff
@@ -83,6 +83,11 @@ if (window.location.hostname !== 'localhost') {
         )
       )
         return null
+
+      if (/\/api\/v1\/(send|gas\/estimate)/.test(event.request?.url ?? '')) {
```

Comment:
seems fine for now and if we see a lot of dupe errors, we could use the url for `gas/estimate` to still group same failing estimation reqs and also the send body for grouped fingerprints

Reply:
Actually very fair point - fingerprinting already contains everything we need as far as inspectability goes - one issue many events.
If/when we manage to get around those 500/520s with missing `Access-Control-Allow-Origin`, we can revisit sends 🙏🏽 but for now I'll close this.